src/mapping/config/MappingConfiguration.cpp

```diff
@@ -324,6 +327,18 @@ MappingConfiguration::ConfiguredMapping MappingConfiguration::createMapping(
   bool    usePETSc        = false;
   RBFType rbfType         = RBFType::EIGEN;
 
+  RBFParameter param;
+  // Check valid combinations for the Gaussian RBF input
+  if (type == VALUE_RBF_GAUSSIAN) {
+    const bool exactlyOneSet = (std::isfinite(supportRadius) && !std::isfinite(shapeParameter)) ||
+                               (std::isfinite(shapeParameter) && !std::isfinite(supportRadius));
```

Comment:
> The suggestions from @fsimonis and mine went a bit in opposite directions. I am also OK with deducing the user's wish automatically. It would still be good to throw a warning if both are configured, however.

@uekerman We safeguard in a strict way against specifying both.

Reply:
:+1: